### PR TITLE
Drop koji repoclosure targets

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: package_manifest.yaml [hunk 1]
```diff
@@ -923,88 +923,11 @@ repoclosures:
...
-          - el8-puppet-7
-          - "el8-foreman-{{ foreman_version }}-staging"
-          - "el8-foreman-plugins-{{ foreman_version }}-staging"
-          - "el8-candlepin-{{ candlepin_version }}-staging"
+          - "el8-candlepin-{{ candlepin_version }}"
     foreman-client-staging-repoclosure-el9:
       repoclosure_target_repos:
         el9:
-          - "el9-foreman-client-{{ foreman_version }}-staging"
+          - "el9-foreman-client-{{ foreman_version }}"
```
Comment (evgeni) [inline package_manifest.yaml:930]: I must admit I liked the `-staging` suffix, as it indicated correctly that we're not looking at the prod repos, but staging.

Not going to block on this tho :)

**Review by evgeni — APPROVED** [review]

File: package_manifest.yaml (hunk 1 above)
Comment (ehelms) [reply package_manifest.yaml:930]: I liked it too... I just assumed that others would want a clean revert back to the way things were with koji gone. I'd be happy to keep the staging nomenclature.

Comment (ekohl) [reply package_manifest.yaml:930]: :+1: on keeping staging explicitly named.

**Comment by ehelms** [comment]
Updated to use staging nomenclature for repoclosure everywhere.

**Review by evgeni — APPROVED** [review]

**Pull request merged**